app.py

```diff
@@ -39,8 +39,9 @@
 # Create databases, if databases exists doesn't issue create
 # For schema changes, run "flask db migrate"
 from models import Restaurant, Review
-db.create_all()
-db.session.commit()
+with app.app_context():
```

Comment:
I had a different fix for the other Flask sample:
https://github.com/pamelafox/a-msdocs-flask-postgresql-sample-app-azd/commit/9f7cf103abb89093592be2888c600ceb46ff0ca1

In that repo, we expect the user to run `flask db upgrade`, so `db.create_all()` is not needed here. From what I understand, if we're using Flask-migrate, it's better practice to run `flask db upgrade`. (It also seems more production-level ready)

Reply:
Ah, forgot that. Let me see I can use that. Thanks.

Reply:
@pamelafox - I'm not having look remove those lines and following your changes.

Reply:
Okay, think I figured it out and I can bring this app.py to be closer to what you have in the link above.